net/oui/Portfile

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+revision            0
+
+version             0.1.8
+name                oui
+
+homepage            https://github.com/thatmattlove/oui
+
+categories          net
+
+description         MAC Address CLI Toolkit
+
+long_description    Look up MAC Address vendors and convert MAC Addresses \
+                    to other formats — offline and at the console.
+
+maintainers         {stunninglyclear.com:matt @thatmattlove} openmaintainer
+
+license             BSD-3-Clause-Clear
+
+github.setup        thatmattlove ${name} ${version} v
+github.tarball_from releases
+
+distname            ${name}_${version}_${os.platform}_${os.arch}64
+
+checksums               ${name}_${version}_darwin_amd64.tar.gz \
+                        sha256  cd26e80c45569ce5aced00892f7226b84dbe25b4a372c74fcd45df5427264fba \
+                        rmd160  997ded2bbb555f3a2d7baedb5ba83c331607f08b \
+                        size    5086101 \
+                        ${name}_${version}_darwin_arm64.tar.gz \
+                        sha256  45cf5925aa6d51d5edf8a1915f3d58b841fa97de3b0ed18273326080c959dc59 \
+                        rmd160 2a3f3c440182b70623cc510764bea709541ff93b \
+                        size    4924616
```

Comment:
You've only provided checksums for arm64 and x86_64 binaries, so you'll need to add the line `supported_archs arm64 x86_64` somewhere so that users on other architectures get a better error message.

The arm64 rmd160 line is not indented the same as the others.